README.md

```diff
@@ -137,14 +137,15 @@ Warning: This is not a security feature. Fields can still easily be rearranged b
 
 #### Advanced Obfuscation
 
-For more advanced obfuscation, you can pass in a custom obfuscation strategy.
-Polo will take in a lambda that can be used to transform sensitive data.
+For more advanced obfuscation, you can pass in a custom obfuscation strategy. Polo will take in a lambda that can be used to transform sensitive data. 
+
+To target a column within a specific table an SQL selector may be used as the obfuscate key.
```

Comment:
can we add a line that clarifies the difference between regular fields vs namespaced ones?
i.e. a naked field will by applied across every table, vs a namespaced one only being applied to that specific table.